### PR TITLE
Milestone 4: Add quote component

### PR DESCRIPTION
**Pull request by prg-04**
### I have done the following to reach this milestone:

- [ ] Created a new component for displaying a quote
- [ ] Fetch data inside of the new component
- [ ] Show a loading state
- [ ] Show an error state

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @prg-04, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Able To Fetch Quotes :heavy_check_mark: 
`useEffect` Hook Has Been Used Correctly :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 
Capitalized Commit Messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

### Due to insufficient files changed I will leave some comments here

- [ ] Please remove the link to the `FAQ` section in the table of contents so as to have a readme file with no broken links
image

- [ ] Kindly fix the link to the `Deployment` subsection of your readme file so that when a user clicks on the link in the table of contents they are taken to the `Deployment` subsection
image

- [ ] Kindly fix the links to your `LinkedIn` and `Twitter` profiles aas they are links which do no exist
image
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: Good job fixing the links for your `LinkedIn & Twitter` handle however there are still some broken links in your readme file which the previous reviewer requested changes for in this [review link](https://github.com/prg-04/math-magician/pull/4#pullrequestreview-1572125427)
- [ ] You currently do not have an faq section in your readme so I kindly suggest that you remove this header from your list of headers at the top of your readme file
faq
- [  ] The deployment link in your readme headers as shown below once clicked does not take you to the deployment section, please kindly fix this link as well as requested by the previous reviewer.
dep

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @prg-04 , 



Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @prg-04 , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Well written readme ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] All requested changes have been implemented ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**